screens/SignupScreen.styled.ts

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components/native';
+
+import Colors from '../theme/Colors';
+
+const StyledContainer = styled.View`
+  flex: 1;
+  flex-direction: column;
```

Comment:
Unnecessary `flex-direction` and `justify-content` properties set to default values.

Reply:
You right, I will remove it.